### PR TITLE
set maven libs according to consumers

### PR DESCRIPTION
**Pull request by jtnord**
https://groups.google.com/g/jenkinsci-dev/c/Xua30fqoqZc

downgrade libraries and add dependabot exclusion

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -53,12 +53,16 @@
...
     <gitHubRepo>jenkinsci/${project.artifactId}</gitHubRepo>
     <aether.version>1.1.0</aether.version>
-    <maven.version>3.9.0</maven.version>
+    <maven.version>3.8.1</maven.version> <!-- if we update this all consumers need to be running at least this version -->
     <maven-plugin-tools.version>3.8.1</maven-plugin-tools.version> <!-- extra work needed when https://issues.apache.org/jira/browse/MPLUGIN-449 is fixed -->
     <plexus-containers.version>2.1.1</plexus-containers.version>
     <aether.version>1.1.0</aether.version>
   </properties>
 
+  <prerequisites>
```
Comment (jtnord) [inline pom.xml:62]: not strictly necessary - but as we do not know if we use an api that was only introduced in this version it prevents random blow ups in consumers.

**Review by basil — APPROVED** [review]

**Review by MarkEWaite — APPROVED** [review]
Thanks!

**Review by olamy — APPROVED** [review]

**Comment by basil** [comment]
This isn't the only Maven plugin maintained by the Jenkins core team. For consistency, `maven-jellydoc-plugin`,`maven-stapler-plugin`, `incrementals-maven-plugin`, `access-modifier-checker`, and `bridge-method-injector` need the same treatment.

**Comment by basil** [comment]
For the task to be fully complete, all of the above PRs (including this PR) should be released, and consumers updated to adopt those releases.

**Pull request merged**